cassandra/src/test/java/org/hibernate/ogm/datastore/cassandra/test/mapping/CassandraBooleanCharacterTypeTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.test.mapping;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.hibernate.ogm.datastore.cassandra.type.impl.CassandraTrueFalseType;
+import org.hibernate.ogm.datastore.cassandra.type.impl.CassandraYesNoType;
+import org.hibernate.ogm.model.spi.Tuple;
+import org.hibernate.ogm.type.descriptor.impl.GridValueBinder;
+import org.hibernate.ogm.type.descriptor.impl.GridValueExtractor;
+import org.junit.Test;
+
+/**
+ * Tests that 'yes_no' and 'true_false' types are correctly mapped to single-character strings.
+ *
+ * @author Nicola Ferraro
+ */
+public class CassandraBooleanCharacterTypeTest {
+
+	@Test
+	public void testYesNoTypeMapping() {
+
```

Comment:
Interesting, I never thought of testing it this way :)

Usually we use more integrative means of testing (like the mapping format tests you've seen), but I suppose you did it that way because the required infra for that is part of your other PR? That's cool, if you like you can change it later on, but nothing critical to me.